scripts/render.js

```diff
@@ -0,0 +1,25 @@
+import fs from "fs-extra";
+import glob from "glob";
+import path from "path";
+import { getNameAndSize } from "./util/helpers.js";
+
+const svgPaths = glob.sync(`./dist/**/*.svg`);
+
+export const iconsBySize = () => {
+  const iconsBySize = {};
+  const icons = svgPaths.map((svgPath) => {
+    const data = fs.readFileSync(svgPath, "utf-8");
+    const {size} =  getNameAndSize(svgPath);
+    return {
+      name: path.parse(svgPath).name,
```

Comment:
This will return the name of the svg file (like ` icon_24`), while it's more the parent directory that we'd need for the preview page (like `active-ads`). I tested that this would give us what we need:
```suggestion
      name: path.basename(path.dirname(svgPath)),
```
Wdyt?

<img width="1027" alt="Screenshot 2023-08-25 at 13 20 19" src="https://github.com/warp-ds/icons/assets/41303231/aa4dbb9f-2f4d-4e67-8d41-18483ebdc8a9">

Reply:
oh that was a way cleaner way to do it!

Reply:
That is great, committed 🙌